File: app/src/main/java/com/tm/blplayer/utils/constants/PathConstants.java
package com.tm.blplayer.utils.constants;

import android.os.Environment;

import com.tm.blplayer.BLApplication;

/**
 * @author wutongming
 * @description 路径常量
 * @since 2017/4/17
 */

public class PathConstants {
    /**
     * SDCard路径
     */
    public final static String SDCARD = Environment.getExternalStorageDirectory().getAbsolutePath();

    /**
     * 缓存文件路径
     */
    public final static String CACHE = BLApplication.getContext().getCacheDir().getAbsolutePath();

    /**
     * HTTP缓存文件存储路径
     */
    public final static String CACHE_HTTP = CACHE + "/http/";

    /**
     * Glide缓存文件路径
     */
    public final static String CACHE_GLIDE = CACHE + "/glide/";

    /**
     * SDCard文件存储路径
     */
    public final static String SDCARD_BL = SDCARD + "/BLPlayer/";
}

File: app/src/main/java/com/tm/blplayer/utils/constants/PathConstants.kt
package com.tm.blplayer.utils.constants

import android.os.Environment
import com.tm.blplayer.BLApplication

/**
 * @author wutongming
 * *
 * @description 路径常量
 * *
 * @since 2017/4/17
 */

object PathConstants {
    /**
     * SDCard路径
     */
    val SDCARD = Environment.getExternalStorageDirectory().absolutePath ?: "/"

    /**
     * 缓存文件路径
     */
    val CACHE = BLApplication.instance?.cacheDir?.absolutePath ?: "/"

    /**
     * HTTP缓存文件存储路径
     */
    val CACHE_HTTP = CACHE + "/http/"

    /**
     * Glide缓存文件路径
     */
    val CACHE_GLIDE = CACHE + "/glide/"

    /**
     * SDCard文件存储路径
     */
    val SDCARD_BL = SDCARD + "/BLPlayer/"
}
